src/main/java/org/jabref/logic/journals/LwtaAbbreviationRepository.java

```diff
@@ -0,0 +1,209 @@
+package org.jabref.logic.journals;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+public class LwtaAbbreviationRepository {
+
+    private final Map<String, LwtaAbbreviation> lwtaToAbbreviationObject;
+
+    // incomplete list
```

Comment:
What is that for an "incomplete" list. What should one do to add words? Either state that or remove the comment.